androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -119,6 +204,40 @@ fun EnhancedVideoPlayer(
                 }
             }
         )
+        Row(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(
+                    if (rewindState || forwardState) {
+                        Color.Black.copy(alpha = 0.5f)
```

Comment:
This should be the same as the player control background, shouldn't it? So let's move it to a const variable to keep the pattern